qiskit/result/utils.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Utility functions for working with Results."""
+
+from functools import reduce
+from re import match
+from qiskit.exceptions import QiskitError
+
+
+def marginal_counts(counts, indices=None, pad_zeros=False):
+    """Marginalize counts from an experiment over some indices of interest.
+
+    Args:
+        counts (dict): Counts of some measurement results, to be marginalized.
```

Comment:
```suggestion
        counts (Dict): Counts of some measurement results, to be marginalized.
```

Reply:
this is following the rest of docstrings in qiskit, which is not quite the same as the pulse package.